packages/wrangler/e2e/unenv-preset/preset.test.ts

```diff
@@ -125,7 +121,36 @@ const testConfigs: TestConfig[] = [
 			},
 		},
 	],
-].flat();
+	// node:fs and node:fs/promises
+	[
+		{
+			name: "fs disabled by date",
+			compatibilityDate: "2025-07-26",
+			compatibilityFlags: ["experimental"],
+			expectRuntimeFlags: {
+				enable_nodejs_fs_module: false,
+			},
+		},
+		// TODO: add a config when fs is enabled by default (date no set yet)
```

Comment:
As the other TODO, are the todos in this file for now?

Reply:
TODO are for when a default enabled is set (see "date no set yet")